bukkit/src/main/java/net/playlegend/legendshowlog/bukkit/ShowLogCommand.java

```diff
@@ -29,22 +29,18 @@
 
   private static final long MAX_FILE_LENGTH = 5 * 1024 * 1024L;
   private static final ExecutorService EXECUTOR_SERVICE = Executors.newSingleThreadExecutor();
+  private static final Gson GSON = new Gson();
```

Comment:
only constants should be written in uppercase. https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names
We did this wrong in multiple places. Therefore could you change GSON to gson and EXECUTOR_SERVICE to executorService?